src/codegen/ast_interpreter.cpp

```diff
@@ -69,7 +69,8 @@ union Value {
 
 class ASTInterpreter {
 public:
-    typedef llvm::DenseMap<InternedString, Box*> SymMap;
+    typedef llvm::DenseMap<InternedString, int> SymMap;
+    typedef std::vector<Box*> SymVector;
```

Comment:
Yeah, I added ContiguousMap, and then found a few places where I hadn't converted over, so it was a good change.